compiler/codegen/CodeGenRA.cpp

```diff
@@ -490,7 +493,7 @@ OMR::CodeGenerator::allocateSpill(int32_t dataSize, bool containsCollectedRefere
          }
 
      if (
-         (spill && self()->comp()->getOption(TR_TraceRA) && !performTransformation(self()->comp(), "O^O SPILL TEMPS: Reuse spill temp %s\n", self()->getDebug()->getName(spill->getSymbolReference()))))
+         (spill && self()->comp()->getOption(TR_TraceRA) && !performTransformation(self()->comp(), "%sReuse spill temp %s\n", OPT_DETAILS_SPILL_TEMPS, self()->getDebug()->getName(spill->getSymbolReference()))))
```

Comment:
We can use generic `OPT_DETAILS` since `SPILL TEMPS` doesn't add anything we can't already make out from the transformation message